authentication_document.py

```diff
@@ -45,13 +52,13 @@ def __init__(self, _db, id, title, type, service_description, color_scheme,
                 _db, service_area, place_class=place_class
             )
         else:
-            self.service_area = [place_class.everywhere(_db)], [], []
+            self.service_area = [place_class.everywhere(_db)], {}, {}
```

Comment:
parse_coverage returns a list and two dicts, and this should use the same data types.